org.gnome.Hamster.json

```diff
@@ -0,0 +1,69 @@
+{
+    "app-id": "org.gnome.Hamster",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "41",
+    "sdk": "org.gnome.Sdk",
+    "command": "hamster",
+    "finish-args": [
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--share=ipc",
+        "--filesystem=xdg-documents",
+        "--own-name=org.gnome.Hamster",
+    ],
+    "modules": [
+        {
+            "name": "intltool",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://launchpad.net/intltool/trunk/0.51.0/+download/intltool-0.51.0.tar.gz",
+                    "sha256": "67c74d94196b153b774ab9f89b2fa6c6ba79352407037c8c14d5aeb334e959cd"
+                }
+            ],
+            "cleanup": [ "*" ]
+        },
+        {
+            "name": "python3-hamster-dependencies",
+            "buildsystem": "simple",
+            "ensure-writable": [
+                "easy-install.pth",
+                "setuptools.pth"
+            ],
+            "build-commands": [
+                "pip3 install --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} dbus-python pyxdg"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "url": "https://files.pythonhosted.org/packages/62/7e/d4fb56a1695fa65da0c8d3071855fa5408447b913c58c01933c2f81a269a/dbus-python-1.2.16.tar.gz",
+                    "sha256": "11238f1d86c995d8aed2e22f04a1e3779f0d70e587caffeab4857f3c662ed5a4"
+                },
+                {
+                    "type": "file",
+                    "url": "https://files.pythonhosted.org/packages/ea/13/de39ddf4f9f9cea0c7684cd54a50d79c97ea99c9f6aed798fd13d0bd4609/pyxdg-0.27-py2.py3-none-any.whl",
+                    "sha256": "2d6701ab7c74bbab8caa6a95e0a0a129b1643cf6c298bf7c569adec06d0709a0"
+                }
+            ]
+        },
+        {
+            "name": "hamster",
+            "buildsystem": "simple",
+            "builddir": true,
+            "prefix": "/app",
+            "build-commands": [
+                "./waf configure --prefix=${FLATPAK_DEST}",
+                "./waf build",
+                "./waf install"
+            ],
+            "sources": [
+                {
+                        "type": "archive",
+                        "url": "https://github.com/rabin-io/hamster/archive/refs/tags/release-3.0.2.tar.gz",
```

Comment:
Sorry, this may already have been answered but is there a reason this doesn't point to the upstream source release but to your fork instead?